docs/resources/meeting_conferences.md

```diff
@@ -0,0 +1,526 @@
+
+  -> At least one of `phone`, `email` and `sms` must be set.
+
+<a name="conference_cycle_params"></a>
+The `cycle_params` block supports:
+
+* `cycle` - (Optional, String) Specifies the period type. The valid values are as follows:
+  + **Day**
+  + **Week**
+  + **Month**
+
+* `pre_remind` - (Optional, Int) Specifies the number of days for advance conference notice.
+  The valid value is range from `0` to `30`, defaults to `1`.
+
+* `start_date` - (Optional, String) Specifies the start date of the recurring conference.
+  The format is `YYYY-MM-DD`.
+
+* `end_date` - (Optional, String) Specifies the end date of the recurring conference.
+  The format is `YYYY-MM-DD`.
+
+* `interval` - (Optional, Int) Specifies the cycle interval.
+  For different `cycle` types, the value range of interval are as follows:
+  + **Day**: Means that it will be held every few days, and the valid value is range from `1` to `15`.
+  + **Week**: Means that it is held every few weeks, and the valid value is range from `1` to `5`.
+  + **Month**: Means every few months, the value range is `1` to `3`.
+
+* `points` - (Optional, String) Specifies the conference point in the cycle. Only valid by **Week** and **Month**.
+  For different `cycle` types, the value range of elements are as follows:
+  + **Week**: The valid value is range from `0` to `6`. The **0** means Sunday, **6** means Saturday.
+  + **Month**: The valid range for the elements is `1` to `31`. If the value does not exist in the current month, the
+  value means the end of the month.
+
+<a name="conference_configuration"></a>
+The `configuration` block supports:
+
+* `is_send_notify` - (Optional, Bool) Specifies whether to send conference email notification.
+
+* `is_send_sms` - (Optional, Bool) Specifies whether to send conference SMS notification.
+
+* `is_send_calendar` - (Optional, Bool) Specifies whether to send conference calendar notifications.
+
+* `is_auto_mute` - (Optional, Bool) Specifies whether the soft terminal is automatically muted when the guest joins the
+  conference.
+
+* `is_hard_terminal_auto_mute` - (Optional, Bool) Specifies whether the guest joins the conference, whether the hard
+  terminal is automatically muted.
+
+* `is_guest_free_password` - (Optional, Bool) Specifies whether the guest is password-free (only valid for random
+  conferences).
+
+* `callin_restriction` - (Optional, Int) Specifies the range to allow incoming calls.
+  + **0**: All users.
+  + **2**: Users within the enterprise.
+  + **3**: The invited user.
+
+* `allow_guest_start` - (Optional, Bool) Specifies whether to allow guests to start conferences (only valid for random
+  ID conferences).
+
+* `guest_password` - (Optional, String) Specifies the guest password (pure number 4-16 digits long).
+
+* `prolong_time` - (Optional, Int) Specifies the Automatically extend duration (recommended value range is 0-60).
```

Comment:
it's limited to 0-60 in the code, not recommended

Reply:
done.